lecture/views.py

```
@@ -2,48 +2,59 @@
 from django.contrib.auth.decorators import login_required
 from django.contrib import messages
 from django.urls import reverse
-from django.http import HttpRequest, HttpResponse
-from .forms import LectureForm, LecturePDFForm
-from .models import Lecture
-from courses.models import Courses
 from django.views.decorators.http import require_http_methods
+from django.http import HttpRequest, HttpResponse, HttpResponseForbidden, Http404
```

Ruff (0.8.2)

`django.http.Http404` imported but unused

Remove unused import: `django.http.Http404`

(F401)